connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/internal/network/PingResponse.kt

```diff
@@ -0,0 +1,48 @@
+package com.theoplayer.android.connector.uplynk.internal.network
```

Comment:
`PingResponse` is returned in the `UplynkListener.onPingResponse`, so it should not be in the internal package.

Reply:
Moved